Mathlib/Data/Finsupp/Single.lean

```diff
@@ -101,7 +100,9 @@ theorem support_single_ne_zero (a : α) (hb : b ≠ 0) : (single a b).support =
   if_neg hb
 
 theorem support_single_subset : (single a b).support ⊆ {a} := by
-  classical show ite _ _ _ ⊆ _; split_ifs <;> [exact empty_subset _; exact Subset.refl _]
+  classical
```

Comment:
I'm not sure if mathlib style is to indent subsequent lines, but I don't mind it either, so :shrug:

Reply:
Good catch, thanks